include/nfl/prng/crypto_stream_salsa20.h

```diff
@@ -1,11 +1,16 @@
 #ifndef CRYPTO_STREAM_H
 #define CRYPTO_STREAM_H
 
+#ifdef __cplusplus
+#include <string>
+#endif
+
+namespace nfl
+{
```

Comment:
That's a non-sense too, if you use namespace out of the __cplusplus guard, as namesapce is a C++ feature :-/